### ISSUE TITLE
i7-7700k, generated SSDT.aml unable to reduce max turbo frequency

### ISSUE THREAD

**Issue by CarloApri** [Piker-Alpha/ssdtPRGen.sh#400]
I used the Beta of ssdtPRGen.sh to generate the SSDT (./ssdtPRGen.sh -turbo 4100) with no warnings, placed it in patched folder, but I still peak at 4.4GHz with Intel Gadget.
I was told that my SMBIOS (iMac 18,3) uses Intel HWP/ISS, so I tried to use a different SMBIOS, not HWP compatible (17,1), but with no success. I don't have NULLCPUPowerManagement.kext, I have Plugin Type = 1 in my config.plist (Acpi/SSDT) with drop tables, and I have HWP Enable "false".

**Comment by 0x6d7a** [comment]
Hi bro, did you manage to solve the problem?

**Comment by CarloApri** [comment]
> Hi bro, did you manage to solve the problem?

No, I didn’t. My problem was that fans started to spin too much/loud because whenever I used the hackintosh, whether opening a program or just browsing, that would made my processor frequency peak, so I changed the fan curve in the BIOS and that was it, temperatures are still fine after all, slightly higher than win10. One could also disable intel turbo in the BIOS to prevent the processor to reach the turbo peak frequencies, maybe there is a software solution to do that on macOS, not sure.
